File: src/main/java/com/talenco/tasksystem/service/UserService.java
package com.talenco.tasksystem.service;

import com.talenco.tasksystem.entity.User;

import java.util.List;

public interface UserService {

    List<User> getAll();

    User getOne(String username);

    void insert(User user);

    void update(User user);

    void delete(Long[] ids);


}

File: src/main/webapp/js/service/userService.js
app.service('userService',function($http){

    // 用户列表
    this.getAll=function(){
        return $http.get('../UserController/getAll');
    }
    // 用户查询
    this.getOne=function(username){
        return $http.get('../UserController/getOne?username='+username);
    }
     // 用户查询
    this.getManager=function(){
        return $http.get('../UserController/getManager');
    }

    // 新建用户
    this.insert=function(entity){
        return $http.post('../UserController/insert',entity);
    }
    // 删除用户
    this.dele=function(selectIds){
        if(confirm("你确认要删除么")) {
            return $http.get('../UserController/delete?ids=' + selectIds);
        }
    }
    // 更新用户信息
    this.update=function(entity){
        return $http.post('../UserController/update',entity);
    }
})